scipy/stats/_qmc.py

```diff
@@ -914,13 +910,51 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    strength : {1, 2}, optional
+        Strength of the LHS. ``strength=1`` produces a plain LHS while
+        ``strength=2`` produces an orthogonal array based LHS of strength 2
+        [7]_, [8]_. In that case, only ``n=p**2`` points can be sampled,
+        with ``p`` a prime number. It also constrains ``d <= p + 1``.
+        Default is 1.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    When LHS is used for integrating a function :math:`f` over :math:`n`,
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    With a LHS of :math:`n` points, the variance of the integral is always
+    lower than plain MC on :math:`n-1` points [3]_. There is a central limit
+    theorem for LHS on the mean and variance of the integral [4]_, but not
+    necessarily for optimized LHS due to the randomization.
+
+    :math:`A` is called an orthogonal array of strength :math:`t` if in each
+    n-row-by-t-column submatrix of :math:`A`: all :math:`p^t` possible
+    distinct rows occur the same number of times. The elements of :math:`A`
+    are in the set :math:`\{0, 1, ..., p-1\}`, also called symbols.
+    The constraint that `p` must be a prime number is to allow modular
+    arithmetic.
+
+    Strength 1 (plain LHS) brings an advantage over strength 0 (MC) and
+    strength 2 is a useful increment over strength 1. Going to strength 3 is
+    a smaller increment and scrambled QMC like Sobol', Halton are more
+    performant [7]_.
+
+    To create a LHS of strength 2, the orthogonal array :math:`A` is
+    randomized by randomly permuting the symbol values on each column
+    independently. It's the set of symbols which is permuted, not the
+    independent elements. Hence all 0 can becomes 2, 1 become 2, etc.
+    Then, on each column and for a given symbol value, a plain LHS of size
+    ``(p, 1)`` is added. The resulting matrix is finally divided by :math:`p`.
```

Comment:
The code is so much easier to understand. How about:
```suggestion
    Then, for each column `i` and symbol `j`, we add a plain, 
    one-dimensional LHS of size `p` to the subarray where `A[:, i] == j`.
    The resulting matrix is finally divided by :math:`p`.
```

Reply:
Ok sure, I will use the math notation as well then.